### PR TITLE
Link GitHub Check Results to RHTAP UI for pipelinerun details and pip…

### PR DESCRIPTION
**Pull request by jkhelil**
…elinerun logs

Fixes https://issues.redhat.com/browse/PLNSRVCE-1484
-  dynamically configure custom-console-url-pr-details and custom-console-url-pr-tasklog using https://github.com/openshift-pipelines/pipelines-as-code/blob/main/docs/content/docs/install/settings.md

Also linked to https://github.com/openshift/hac-dev/pull/903

### CODE REVIEW

**Comment by jkhelil** [comment]
@piyush-garg @savitaashture

**Comment by jkhelil** [comment]
We do want to deploy it on stagging for testing

File: components/pipeline-service/staging/base/update-tekton-config-pac.yaml [hunk 1]
```diff
@@ -5,5 +5,5 @@
     application-name: Konflux Staging
     custom-console-name: Konflux Staging
     custom-console-url: https://console.dev.redhat.com/preview/application-pipeline
-    custom-console-url-pr-details: https://console.dev.redhat.com/preview/application-pipeline
-    custom-console-url-pr-tasklog: https://console.dev.redhat.com/preview/application-pipeline
+    custom-console-url-pr-details: https://console.dev.redhat.com/preview/application-pipeline/ns/{{ namespace }}/pipelienrun/{{ pr }}
+    custom-console-url-pr-tasklog: https://console.dev.redhat.com/preview/application-pipeline/ns/{{ namespace }}/pipelienrun/{{ pr }}/logs/{{ task }}
```
Comment (ramessesii2) [inline components/pipeline-service/staging/base/update-tekton-config-pac.yaml:9]:
```suggestion
    custom-console-url-pr-tasklog: https://console.dev.redhat.com/preview/application-pipeline/ns/{{ namespace }}/pipelinerun/{{ pr }}/logs/{{ task }}
```
Got a typo.

**Review by ramessesii2 — COMMENTED** [review]
A typo it seems in all the annotations template patterns

File: components/pipeline-service/staging/base/update-tekton-config-pac.yaml (hunk 1 above)
Comment (jkhelil) [reply components/pipeline-service/staging/base/update-tekton-config-pac.yaml:9]: Nice catch !!!!

Comment (jkhelil) [reply components/pipeline-service/staging/base/update-tekton-config-pac.yaml]: fixed

**Review by ramessesii2 — APPROVED** [review]
/lgtm

**Comment by ramessesii2** [comment]
Hey @jkhelil, it seems we need approval from @Roming22 to merge it.
Also, `appstudio-hac-e2e-tests` is optional. So, I believe we can get this merge to test it faster.

**Comment by jkhelil** [comment]
@Roming22 do you think we can force merge this one please ?

**Review by Roming22 — APPROVED** [review]
/lgtm

**Review by Roming22 — CHANGES_REQUESTED** [review]
You forgot to regenerate the deploy.yaml.

File: components/pipeline-service/staging/stone-stage-p01/deploy.yaml [hunk 2]
```diff
@@ -1,2041 +0,0 @@
-apiVersion: v1
```
Comment (Roming22) [inline components/pipeline-service/staging/stone-stage-p01/deploy.yaml:1]: The files were not generated properly, they all seem to be empty :wink:

**Review by Roming22 — CHANGES_REQUESTED** [review]

File: components/pipeline-service/staging/stone-stage-p01/deploy.yaml (hunk 2 above)
Comment (jkhelil) [reply components/pipeline-service/staging/stone-stage-p01/deploy.yaml]: fixed now

**Review by Roming22 — APPROVED** [review]
I will force merge it since it does not impact the CI.

**Pull request merged**